app/models/booking.rb

```diff
@@ -1,6 +1,18 @@
 class Booking < ApplicationRecord
-  validates :status, inclusion: { in: ["Pending", "Accepted", "Declined", "Cancelled"] }
-  has_many :reviews, dependent: :destroy
   belongs_to :user
   belongs_to :garment
+  has_many :reviews, dependent: :destroy
+
+  validates :start_date, presence: true
+  validates :end_date, presence: true
+  validates :status, inclusion: { in: ["Pending", "Accepted", "Declined", "Cancelled"] }
```

Comment:
Just on here - I've done all the controllers based off ['pending, 'confirmed', 'declined', 'cancelled'] if we can just update it caps may be an issue as well :)